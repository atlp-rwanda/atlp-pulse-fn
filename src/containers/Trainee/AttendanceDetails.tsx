/* eslint-disable */
import React from 'react';
import TraineeAttendaceDetails from '../../components/TraineeAttendanceDetails';
import TraineeComments from '../../components/TraineeComments';
import useDocumentTitle from '../../hook/useDocumentTitle';

function AttendanceDetails() {
  useDocumentTitle('Attendance details');
  return (
    <div className="bg-neutral-100 dark:bg-dark-frame-bg">
      <TraineeAttendaceDetails />
      {/* <TraineeComments /> */}
    </div>
  );
}

export default AttendanceDetails;
