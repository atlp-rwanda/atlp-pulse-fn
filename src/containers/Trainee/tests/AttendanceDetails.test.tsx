/* eslint-disable */
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import AttendanceDetails from '../AttendanceDetails';

describe('Attendance Detailed page', () => {
  it('Renders the Attendance Detailed Page ', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <AttendanceDetails />
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
