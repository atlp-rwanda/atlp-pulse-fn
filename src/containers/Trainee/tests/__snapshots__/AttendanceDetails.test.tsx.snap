// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attendance Detailed page Renders the Attendance Detailed Page  1`] = `
<div
  className="bg-neutral-100 dark:bg-dark-frame-bg"
>
  <div>
    <div
      className="bg-neutral-100 dark:bg-dark-frame-bg md:flex sm:hidden flex-col justify-start items-center"
    >
      <table
        className="dark:bg-dark-bg shadow-lg py-8 rounded-md"
      >
        <thead>
          <tr
            className="text-gray-300  bg-gray-100  dark:bg-dark-tertiary"
          >
            <th
              className="lg:py-3 md:py-2 lg:px-3 md:px-2  text-left  text-[#6B7280] dark:text-dark-text-fill "
            >
              Session
            </th>
            <th
              className="lg:py-3 md:py-2 lg:px-10 md:px-7 text-center text-[#6B7280] dark:text-dark-text-fill"
            >
              Record
            </th>
            <th
              className="lg:py-3 md:py-2 text-left text-[#6B7280] dark:text-dark-text-fill"
            >
              Comment
            </th>
            <th
              className="lg:py-3 md:py-2"
            />
          </tr>
        </thead>
        <tbody
          className=" text-center "
        >
          <tr
            className="text-light-text dark:text-dark-text-fill bg-light-bg dark:bg-dark-bg "
          >
            <td
              className="lg:py-10 md:py-0 px-10 text-left  "
            >
              Demo
            </td>
            <td
              className="py-3 "
            >
              1
            </td>
            <td
              className="lg:py-6 md:py-3 text-start lg:text-sm"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
            </td>
            <td
              className="py-3 px-8"
            >
              <button
                className="btn primary sm px-4 py-0 text-sm"
                disabled={false}
                type="button"
              >
                Reply
              </button>
            </td>
          </tr>
          <tr
            className="text-black dark:text-dark-text-fill bg-gray-100 dark:bg-dark-tertiary "
          >
            <td
              className="py-10 px-10 text-left"
            >
              Standup
            </td>
            <td
              className="py-3 "
            >
              1
            </td>
            <td
              className="py-3  text-start text-sm"
            >
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam
            </td>
            <td
              className="py-3 "
            >
              <button
                className="btn primary sm px-4 py-0 text-sm"
                disabled={false}
                type="button"
              >
                Reply
              </button>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div
      className="sm:flex sm:flex-col md:hidden justify-center items-center bg-light-bg dark:bg-dark-frame-bg dark:text-white  text-black"
    >
      <div
        className="flex flex-col justify-center items-start w-full py-4 px-4 ml-4 mt-4"
      >
        <h2
          className="font-semibold mt-12 "
        >
           
          Sprint 1
        </h2>
        <h3
          className="font-medium text-left text-[#6B7280] dark:text-white "
        >
          June 20, 2022 - June 24, 2022
        </h3>
      </div>
      <div
        className="flex flex-col justify-center items-center w-11/12 border shadow rounded-lg dark:border-dark-frame-bg bg-white  dark:bg-dark-bg pb-4 mt-4 pt-16"
      >
        <table
          className=" rounded-lg w-7/12 h-[70%] overflow-hidden  md:shadow mt-[-40px]"
        >
          <thead>
            <tr
              className="text-gray-300 dark:text-white  bg-gray-100 dark:bg-dark-tertiary "
            >
              <th
                className="py-2 px-3 text-center text-[#6B7280] dark:text-dark-text-fill "
              >
                Session
              </th>
              <th
                className="py-2 px-10 text-center text-[#6B7280] dark:text-dark-text-fill"
              >
                Demo
              </th>
            </tr>
          </thead>
          <tbody
            className=" text-center"
          >
            <tr
              className="text-light-text bg-light-bg dark:bg-dark-bg dark:text-dark-text-fill  "
            >
              <td
                className="py-3 px-10"
              >
                Record
              </td>
              <td
                className=" "
              >
                1
              </td>
            </tr>
            <tr
              className="text-black bg-gray-100 dark:bg-dark-tertiary dark:text-dark-text-fill  "
            >
              <td
                className="py-10 px-10 text-left"
              >
                Comments
              </td>
              <td
                className="py-3  text-start text-sm"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </td>
            </tr>
          </tbody>
        </table>
        <button
          className="px-4 py-1 ml-40 mt-4 rounded-md dark:text-dark-text-fill text-center bg-primary text-white  text-sm"
        >
          Reply
        </button>
      </div>
      <div
        className="flex flex-col justify-center items-center w-11/12 border shadow rounded-lg dark:border-dark-frame-bg bg-white dark:bg-dark-bg  pb-4 mt-4 pt-16"
      >
        <table
          className=" rounded-lg w-7/12 h-[70%] overflow-hidden  md:shadow mt-[-40px]"
        >
          <thead>
            <tr
              className="text-gray-300 dark:text-dark-text-fill bg-gray-100 dark:bg-dark-tertiary"
            >
              <th
                className="py-2 px-3 text-center text-[#6B7280] dark:text-dark-text-fill  "
              >
                Session
              </th>
              <th
                className="py-2 px-10 text-center text-[#6B7280] dark:text-dark-text-fill"
              >
                Standup
              </th>
            </tr>
          </thead>
          <tbody
            className=" text-center"
          >
            <tr
              className="text-light-text bg-light-bg dark:bg-dark-bg dark:text-dark-text-fill "
            >
              <td
                className="py-3 px-10"
              >
                Record
              </td>
              <td
                className=" "
              >
                1
              </td>
            </tr>
            <tr
              className="text-black bg-gray-100 dark:bg-dark-tertiary dark:text-dark-text-fill  "
            >
              <td
                className="py-10 px-10 text-left"
              >
                Comments
              </td>
              <td
                className="py-3  text-start text-sm"
              >
                Lorem ipsum dolor sit amet, consectetur adipiscing elit
              </td>
            </tr>
          </tbody>
        </table>
        <button
          className="px-4 py-1 ml-40 mt-4 rounded-md dark:text-dark-text-fill text-center bg-primary text-white  text-sm"
        >
          Reply
        </button>
      </div>
    </div>
  </div>
</div>
`;
